image/cli/install-ansible/install-ansible-collections.sh

```diff
@@ -2,7 +2,15 @@
 
 ansible-galaxy collection install -r /tmp/install-ansible/requirements.yml -p $ANSIBLE_COLLECTIONS_PATH
 
-if [[ -e /tmp/install-ansible/ibm-mas_devops.tar.gz ]]
-then ansible-galaxy collection install /tmp/install-ansible/ibm-mas_devops.tar.gz -p $ANSIBLE_COLLECTIONS_PATH
-else ansible-galaxy collection install ibm.mas_devops
-fi
+git clone -b MASCORE-24 https://github.com/yuvraj-vansure/ansible-devops.git
```

Comment:
this change in this file needs to be removed

Reply:
yes, has added for testing, will remove